src/components/preferences/Webhooks.js

```diff
@@ -82,11 +88,15 @@ export default function Webhooks(props) {
                         name="webhook.type.type"
                         id={buildID(baseId, ids.WEBHOOK_TYPES_SELECT)}
                         label={t("type")}
-                        required
                         select
                         variant="outlined"
                         size="small"
                         component={FormTextField}
+                        validate={(value) => {
+                            if (values?.webhook?.url && !value) {
+                                return t("webhookTypeError");
+                            }
+                        }}
                     >
```

Comment:
Finally got the validation to work by removing `required` and adding field level validation.